### PR TITLE
Show alerts in flat (not 3D) style #7059

### LINKED ISSUE #7059: Show alerts in flat (not 3D) style
To avoid alerts looking like expandable panels, we can use a flat (not 3D style) for alerts.

The current style:
image

Flat style:
image
image

### PR DESCRIPTION
**Pull request by saching97**
Fixes #7059 

**Outline of Solution**
None of the alerts require expansion of any sort. So I updated the teammatesCommon CSS file so that the alerts are visible in a flat manner. The "Bootstrap theme min" modifies the alerts into a 3D view and it's effects are nullified using the teammatesCommon.css.

**Screenshots**:

### CODE REVIEW

**Comment by saching97** [comment]
Erm, this is confusing; the build shows an error in:

 src/main/webapp/stylesheets/teammatesCommon.css
 143:33  ✖  Expected single quotes   string-quotes

But I have not touched those lines and only inserted lines as shown:



Any advice?

**Comment by whipermr5** [comment]
Wow that is so weird... might be a bug with stylelint! I found that removing the comments helps.

**Comment by saching97** [comment]
@whipermr5 I was initially worried that it was being triggered due to the single quote in 
`Apply to alert to override Bootstrap's default 3d alerts` but it persisted even after removing the apostrophe.
  
Another thing I noticed - on correcting the error of the 143 line only and keeping the other lines as same, the stylelint ran without errors. Nevertheless I have replaced all double with single quotes and I think it's better that the comment stays for future reference.

File: src/main/webapp/stylesheets/teammatesCommon.css [hunk 1]
```diff
@@ -12,6 +12,18 @@ body {
...
 
+/*
+Apply to alert to override Bootstrap's default 3d alerts
+*/
+.alert {
+    -webkit-box-shadow: none;
+    box-shadow: none;
+}
+
+.alert-warning, .alert-success, .alert-info, .alert-danger {
```
Comment (whipermr5) [inline src/main/webapp/stylesheets/teammatesCommon.css]: I just noticed that our [CSS coding standard](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub) specifies that selectors must be written in separate lines. While there are other violations of this in the file, let's not introduce more. I'll open another issue for that.

**Comment by saching97** [comment]
@whipermr5 Sorry I got a bit clunky in the changes. On a side note, I couldn't find the coding standards in the docs. If you could please reference the parent link, it would be great!

**Comment by whipermr5** [comment]
@saching97 No worries! It's in the [README at the bottom of the main docs folder](https://github.com/TEAMMATES/teammates/tree/master/docs).

**Pull request merged**